src/detectors/LOWQ2/LOWQ2.cc

```diff
@@ -35,18 +35,21 @@ extern "C" {
     std::string tracker_readout = "TaggerTrackerHits";
 
     // Digitization of silicon hits
+    auto TaggerTrackerTimeResolution    = 0.195 * dd4hep::ns;
+    auto TaggerTrackerIntegrationWindow = 5.0   * dd4hep::ns;
```

Comment:
```suggestion
    auto TaggerTrackerIntegrationWindow = 25.0   * dd4hep::ns;
```
Upon further thought I think I've had this wrong, 5 ns is about the analogue response time for a MIP but the minimum digital time over threshold is 25 ns.

Reply:
Please can you remind me where the spreadsheet is so I can amend it there too